packages/@n8n_io/eslint-config/base.js

```diff
@@ -324,7 +324,7 @@ const config = (module.exports = {
 		/**
 		 * https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-cycle.md
 		 */
-		'import/no-cycle': 'error',
+		'import/no-cycle': ['error', { ignoreExternal: false, maxDepth: 3 }],
```

Comment:
`false` is the default, did you mean to set it to `true`?

Reply:
it's inverted for some unclear reasons.

Reply:
What do you mean by this? I'd expect we want to ignore cycles in externals to speed up the lint right?

Reply:
I'd expect the same, but this seems to be [an issue in `eslint-plugin-import`](https://github.com/import-js/eslint-plugin-import/issues/2348)